### PR TITLE
Fix version specific tests for 6.10

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
**Description:**
1. Removing starts_in and ends_in decorators, and tests using ends_in from a master.
2. Removing test_content and test_prep610_upgrade, as these are specific to 6.9 and no longer required for 6.10/master branch.
3. Removing `test_positive_automate_bz1624699`, as `smart_proxy_dynflow_core` service isn't available on 6.10
4. `test_positive_foreman_maintain_service_list_sidekiq` isn't for capsule, so removing capsule decorator.
5. Removing fixture call `setup_install_pkgs` from hotfix tests, as it is not required.
6. Using `setup_subscribe_to_cdn_dogfood` in `setup_packages_lock_tests` to subscribe system for package install tests.

Signed-off-by: Gaurav Talreja <gtalreja@redhat.com>

### CODE REVIEW

**Comment by jameerpathan111** [comment]
@Gauravtalreja1 can you provide test results for this pr?

File: tests/test_service.py [hunk 1]
```diff
@@ -51,34 +51,6 @@ def test_positive_foreman_maintain_service_start(ansible_module):
         assert result["rc"] == 0
 
 
-def test_positive_automate_bz1624699(ansible_module):
-    """Validate correct service restart on dynflowd stopped
-
```
Comment (jameerpathan111) [inline tests/test_service.py:56]: @Gauravtalreja1 why are we deleting this test?

File: tests/test_service.py [hunk 2]
```diff
@@ -285,7 +257,6 @@ def test_positive_fm_service_restart_bz_1696862(setup_bz_1696862, ansible_module
         assert result["rc"] == 0
 
 
-@pytest.mark.capsule
 def test_positive_foreman_maintain_service_list_sidekiq(ansible_module):
```
Comment (jameerpathan111) [inline tests/test_service.py:260]: @Gauravtalreja1 why we are removing this capsule test?

File: tests/test_prep_6_10_upgrade.py [hunk 3]
```diff
@@ -1,30 +0,0 @@
-from testfm.log import logger
-
-
-def test_positive_prep_6_10_upgrade(ansible_module):
-    """Runs content directory preparation for pulp3 migration.
```
Comment (jameerpathan111) [inline tests/test_prep_6_10_upgrade.py:5]: @Gauravtalreja1 can you explain why are we deleting this file.

**Comment by jameerpathan111** [comment]
@Gauravtalreja1 can you add a description for this pr, like what this adds/removes and why we are doing that?

File: tests/test_prep_6_10_upgrade.py (hunk 3 above)
Comment (Gauravtalreja1) [reply tests/test_prep_6_10_upgrade.py:5]: Updated reason in the description.

File: tests/test_service.py (hunk 2 above)
Comment (Gauravtalreja1) [reply tests/test_service.py:260]: Updated reason in the description.

File: tests/test_service.py (hunk 1 above)
Comment (Gauravtalreja1) [reply tests/test_service.py:56]: Updated reason in the description.

**Review by jameerpathan111 — APPROVED** [review]

**Pull request merged**